### ISSUE TITLE
Latest version might be broken with jellyfin 10.8.0 and ssl error

### ISSUE THREAD

**Issue by WilssoN93** [nikarh/jellyfin-plugin-authelia#33]
Hey, Nice idea with this plugin! :)

First of all i couldnt use the 1.0.11.0 version, I get this message. And translated would be 
"This plugin has no settings to configure". 
![image](https://github.com/nikarh/jellyfin-plugin-authelia/assets/38328228/4bcf0b0b-ad97-4292-8e0f-4164933c76f5)
Didnt get the same error when using 1.0.10.0 though

My jellyfin version is: 10.8.10

I get an error when trying to make the authentication request
```
System.Net.Http.HttpRequestException: The SSL connection could not be established, see inner exception.
 ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid because of errors in the certificate chain: PartialChain
```
Seems to be related to my self signed certificate. And since Authelia needs https it might be good to have a choice to choose which certificate you want to use? :)

**Comment by nikarh** [comment]
Hey, I will take a look at that today or tomorrow!

Regarding the encryption, personally, I use a LetsEncrypt certificate with a custom domain, but it makes sense to have a configuration option for a custom CA root.
Until it is done, it should be possible to add a root CA to the trust store on your host/container

**Comment by nikarh** [comment]
Hey,

I fixed the build for 10.8.0 and added a textarea configuration where you can paste your root ca PEM. If you think that the approach of having the ca PEM as an fs path would be better (e.g. like an [ldap](https://github.com/jellyfin/jellyfin-plugin-ldapauth) plugin does), please let me know.

https://github.com/nikarh/jellyfin-plugin-authelia/releases/tag/1.0.12

**Comment by WilssoN93** [comment]
Great this worked, Thanks! Is there a way for groups/roles to work through the authelia rest api aswell?
